backend/service/k8s/k8s.go

```diff
@@ -139,8 +139,9 @@ func (s *svc) Clientsets(ctx context.Context) ([]string, error) {
 	return ret, nil
 }
 
-func (s *svc) GetK8sClientset(clientset string) (ContextClientset, error) {
-	return s.manager.GetK8sClientsetDirect(clientset)
+func (s *svc) GetK8sClientset(ctx context.Context, clientset string) (ContextClientset, error) {
```

Comment:
Can we optionally take cluster and namespace also as arguments to this function?